website/docs/r/datadog_monitor_sso_configuration.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor_sso_configuration"
+description: |-
+  Manages SingleSignOn on the datadog Monitor.
+---
+
+# azurerm_datadog_monitor_sso_configuration
+
+Manages SingleSignOn on the datadog Monitor.
+
+## Example Usage
+
+### Enabling SSO on monitor
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-datadog"
+  location = "West US 2"
+}
+resource "azurerm_datadog_monitor_sso_configuration" "test" {
+  datadog_monitor_id        = azurerm_datadog_monitor.example.id
+  singlesignon_state        = "Enable"
+  enterprise_application_id = "XXXX"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `datadog_monitor_id` - (Required) The Datadog Monitor Id which should be used for this.
+
+* `singlesignon_state` - (Required) The state of SingleSignOn configuration.
```

Comment:
Can we just change this to a boolean prop called `enabled`?

Reply:
This parameter gives more context, is it okay to add boolean in the description.

Reply:
I'd prefer making it a boolean, which is consistent with the code base.

Reply:
It's an enum in the backend and can have other states as well like Initial.

Reply:
@magodo let me know if this clarifies it.